aspnet/Gantt/Exporting.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
 title: Exporting| Gantt | ASP.NET | Syncfusion
-description: exporting
+description: Learn here about Exporting with the Syncfusion ASP.NET Web Forms Gantt control, its elements, and more.
```

Comment:
Change description - Learn here about exporting support in Syncfusion ASP.NET Web Forms Gantt control, its elements, and more.